app/controllers/application_controller.rb

```diff
@@ -152,5 +152,6 @@ def set_cache_headers
 
   def skip_session!
     request.session_options[:skip] = true
+    expires_in 0, public: true
```

Comment:
Could you create a new method that encompasses `skip_session!` and `expires_in 0` instead of modifying this? Would be clearer. You can call it `mark_cacheable!` or something.